core/src/components/item/item.scss

```diff
@@ -72,6 +72,8 @@
   --color-focused: var(--color);
   --color-hover: var(--color);
   --ripple-color: currentColor;
+  --background-autofill: rgb(232, 240, 254);
```

Comment:
This value is taken from the light-theme Chrome settings. Need to discuss with the team on where global color tokens are located and how to handle theme support.